src/Makefile.am

```diff
@@ -817,6 +817,7 @@ tests_testClpMap_SOURCES = \
 nodist_tests_testClpMap_SOURCES = \
 	tests/stub_HelperChildConfig.cc \
 	tests/stub_SBuf.cc \
+	tests/stub_SquidConfig.cc \
```

Comment:
Hmm... What happens if instead of linking with stub_SquidConfig.cc, we link with SquidConfig.cc itself? I know that the difference between the two files is that the latter also has Config and Config2 definitions. Do those definitions bring more linking problems? It feels odd that a stub file does not define those two globals (and if we add those definitions, this stub file will not be needed!)...